docs/spec_draft.md

```diff
@@ -1420,6 +1421,79 @@ Produces an `output` tensor from a constant `value`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.convert
+
+### Semantics
+
+Performs an element-wise conversion of values similar to an element-wise
+`static_cast` in C++ from one element type to another on `operand` tensor and
+produces a `result` tensor.
```

Comment:
Let's drop the mention of `static_cast` from C++ to keep the specification self-contained.

```
Performs an element-wise conversion from one element type to another on 
`operand` tensor and produces a `result` tensor.
```